src/scheduler/Time/_util.test.js

```diff
@@ -40,9 +40,9 @@ describe("Time utils", function() {
   });
 
   test("timeValuesToday culls times that have already passed", async () => {
-    // this test will fail between midnight and 1 AM
-    const culled_time_values = timeValuesToday(constructedToday, state.time_values);
-    expect(culled_time_values).not.toHaveLength(6);
+    const now = dayjs().hour(13).minute(59).toString();
+    const culled_time_values = timeValuesToday(now, constructedToday, state.time_values);
+    expect(culled_time_values).toHaveLength(3);
```

Comment:
By adding a new parameter representing the current time to the `timeValuesToday` function, we decouple our reliance on current moving time when the test is running. We can hence test a more edgy scenario where we need to discard 3 hours and keep 3 hours. This should get rid of the previous failure too happening between midnight and 1 AM, as we have a pure function now with no time reading side effect.